dashboard/module.properties

```diff
@@ -0,0 +1,8 @@
+Name: PanoramaWeb Dashboard
+ModuleClass: org.labkey.dashboard.DashboardModule
+ModuleDependencies: TargetedMS
+Label: PanoramaWeb Dashboard
```

Comment:
Please flag this module as Postgres only, since I don't see any need for SQLServer support and that will limit the need to test on both DBs